nasa-gibs/worldview
# WV-2910: Add SeaWiFS Chlorophyll layer

## Description

Fixes #WV-2910.

Add SeaWiFS / OrbView-2 Chlorophyll a layer.

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-259.48216626464614,-146.9788209263796,251.57080825209977,97.04258269223337&z=2&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,SEAWIFS_ORBVIEW-2_GAC_Chlorophyll_a,MODIS_Terra_L2_Chlorophyll_A(hidden),VIIRS_SNPP_L2_Chlorophyll_A,S3B_OLCI_Chlorophyll_a,S3A_OLCI_Chlorophyll_a,OCI_PACE_Chlorophyll_a,VIIRS_NOAA20_Chlorophyll_a,MODIS_Aqua_L2_Chlorophyll_A,VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2000-04-30-T14%3A19%3A21Z)
2. Check that you can see imagery for the SeaWiFS layer, layer descriptions reads well, can take an image snapshot of the layer. 

@nasa-gibs/worldview
